File: kata-others/kata-python/yes_it_fizzy.py
import fileinput

def main():
    #for line in fileinput.input():
    #    print(line)
    input_values = "8 16"
    start, end = map(int, input_values.split())
    for i in range(start, end+1):
        fizz = i % 3 == 0
        buzz = i % 5 == 0
        if fizz and buzz: print('fizz')
        elif fizz: print('fizz')
        elif buzz: print('buzz')
        else: print(i)



if __name__ == "__main__":
    main()

File: kata-others/kata-cpp/yes_it_fizzy.cpp
#include <iostream>
#include <string>
#include <sstream>

int main()
{
    std::string input;
    while (std::getline(std::cin, input))
    {
        // split string in start and end
        std::istringstream iss(input);
        std::string start, end;
        iss >> start >> end;

        int start_num = std::stoi(start);
        int end_num = std::stoi(end);
        for (int i = start_num; i <= end_num; ++i)
        {
            bool fizz = i % 3 == 0;
            bool buzz = i % 5 == 0;
            if (fizz && buzz)
                std::cout << "FizzBuzz\n";
            else if (fizz)
                std::cout << "Fizz\n";
            else if (buzz)
                std::cout << "Buzz\n";
            else
                std::cout << i << '\n';
        }
    }
    return 0;
}
